infra/lib/api-stack/sidechain-fhir-converter-connector.ts

```diff
@@ -134,10 +134,10 @@ export function createLambda({
       MAX_TIMEOUT_RETRIES: String(maxTimeoutRetries),
       DELAY_WHEN_RETRY_SECONDS: delayWhenRetrying.toSeconds().toString(),
       ...(config.lambdasSentryDSN ? { SENTRY_DSN: config.lambdasSentryDSN } : {}),
-      ...(config.loadBalancerDnsName ? { API_URL: config.loadBalancerDnsName } : {}),
+      API_URL: apiServiceDnsAddress,
       QUEUE_URL: sourceQueue.queueUrl,
-      DLQ_URL: dlq.queue.queueUrl,
-      CONVERSION_RESULT_QUEUE_URL: conversionResultQueueUrl,
+      DLQ_URL: dlq.queueUrl,
+      CONVERSION_RESULT_QUEUE_URL: destinationQueue.queueUrl,
```

Comment:
There was no need for an additional parameter, as we already receive the destination queue and can get the URL from that.